<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingWidgetLimit</class>
 <widget class="QWidget" name="SettingWidgetLimit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>697</width>
    <height>396</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>30</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_1">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>1# 正:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pos1">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pos1">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>负:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_neg1">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_neg1">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>2# 正:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pos2">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pos2">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>负:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_neg2">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_neg2">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>3# 正:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pos3">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pos3">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_13">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>负:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_neg3">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_neg3">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>4# 正:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pos4">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pos4">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_14">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>负:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_neg4">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_neg4">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>5# 正:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pos5">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pos5">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_15">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>负:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_neg5">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_neg5">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_6">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>6# 正:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_pos6">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_pos6">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_16">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
       <property name="text">
        <string>负:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_neg6">
       <property name="font">
        <font>
         <pointsize>30</pointsize>
        </font>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_neg6">
       <property name="text">
        <string>当前</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_pos1</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>315</x>
     <y>48</y>
    </hint>
    <hint type="destinationlabel">
     <x>316</x>
     <y>2</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_pos2</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>323</x>
     <y>125</y>
    </hint>
    <hint type="destinationlabel">
     <x>353</x>
     <y>83</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_pos3</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>365</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_pos4</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>350</x>
     <y>281</y>
    </hint>
    <hint type="destinationlabel">
     <x>328</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_pos5</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>348</x>
     <y>351</y>
    </hint>
    <hint type="destinationlabel">
     <x>325</x>
     <y>319</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_pos6</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>351</x>
     <y>437</y>
    </hint>
    <hint type="destinationlabel">
     <x>319</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_neg1</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>661</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>666</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_neg2</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>657</x>
     <y>123</y>
    </hint>
    <hint type="destinationlabel">
     <x>684</x>
     <y>86</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_neg3</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>674</x>
     <y>208</y>
    </hint>
    <hint type="destinationlabel">
     <x>691</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_neg4</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>664</x>
     <y>292</y>
    </hint>
    <hint type="destinationlabel">
     <x>646</x>
     <y>241</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_neg5</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>647</x>
     <y>373</y>
    </hint>
    <hint type="destinationlabel">
     <x>638</x>
     <y>320</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_neg6</sender>
   <signal>clicked()</signal>
   <receiver>SettingWidgetLimit</receiver>
   <slot>SetToCurrentPosition()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>673</x>
     <y>453</y>
    </hint>
    <hint type="destinationlabel">
     <x>687</x>
     <y>401</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SetToCurrentPosition()</slot>
 </slots>
</ui>
